engine/apps/alerts/models/channel_filter.py

```diff
@@ -83,7 +84,7 @@ class ChannelFilter(OrderedModel):
 
     created_at = models.DateTimeField(auto_now_add=True)
 
-    FILTERING_TERM_MAX_LENGTH = 1024
+    FILTERING_TERM_MAX_LENGTH = os.getenv("FILTERING_TERM_MAX_LENGTH", 1024)
```

Comment:
This should not be configurable by env var.  If it changes from one state to another after the database has been initialized then a database migration is required.  The field should be changed to a TextField instead of fixed length CharField, FILTERING_TERM_MAX_LENGTH should be removed and references to it should reference the common JINJA_TEMPLATE_MAX_LENGTH in settings.

Reply:
Does this apply only to FILTERING_TERM_MAX_LENGTH or to other changes too?

Reply:
JINJA_TEMPLATE_MAX_LENGTH = os.getenv("JINJA_TEMPLATE_MAX_LENGTH", 50000)
JINJA_RESULT_TITLE_MAX_LENGTH = os.getenv("JINJA_RESULT_TITLE_MAX_LENGTH", 500)
JINJA_RESULT_MAX_LENGTH = os.getenv("JINJA_RESULT_MAX_LENGTH", 50000)
tit's possible edit vars  on top withiout migration?

Reply:
@mderynck

Reply:
Yes the other ones should be ok to change without needing a database migration.

Reply:
@mderynck i cancel commit with FILTERING_TERM_MAX_LENGTH, pls add other changes